### PR TITLE
various misc fixes for the postfix poller

### PR DESCRIPTION
**Pull request by VVelox**
https://github.com/librenms/librenms/blob/master/includes/polling/applications/postfix.inc.php

~~Move $msefl to be before $sce as it is in the current poller.~~

Also a few misc. changes to better process a few items.

### CODE REVIEW

File: snmp/postfixdetailed [hunk 1]
```diff
@@ -500,10 +507,10 @@ my $data=$received."\n".
 	$rardnf."\n".
 	$rarnfqa."\n".
 	$iuscp."\n".
+	$msefl."\n".
```
Comment (laf) [inline snmp/postfixdetailed]: Why is this moved further up? Doesn't it mean this will now be broken for existing users?

Comment (VVelox) [inline snmp/postfixdetailed]: Yes, because the recently committed PR broke it. This syncs it to that.

https://github.com/librenms/librenms/pull/6173

Comment (laf) [inline snmp/postfixdetailed]: It's much easier to fix in the main repo otherwise users have to re-deploy the application

Comment (VVelox) [inline snmp/postfixdetailed]: Unless we are pulling librenms/librenms#6173 completely that has already happened.

**Comment by laf** [comment]
Main repo pr has been reverted so this is no longer needed

**Comment by VVelox** [comment]
Okay moved $mself back to the end and done some other minor cleanup and corrections.

Nothing that breaks current installs.
